docs/en/latest/getting-started.md

```diff
@@ -115,55 +117,74 @@ We receive a JSON response when we send the request:
 }
 ```
 
-## Pre-Requisites
+## Install Apache APISIX
 
-Before you jump ahead, make sure that you have your machine setup with these tools.
+To get started, we will install Apache APISIX with Docker and enable the [Admin API](./admin-api.md).
 
-- [Docker](https://www.docker.com/) and [Docker Compose](https://docs.docker.com/compose/).
+First clone the [apisix-docker](https://github.com/apache/apisix-docker) repository:
 
-- [curl](https://curl.se/docs/manpage.html) for testing the API. Alternatively, you can use tools like [Hoppscotch](https://hoppscotch.io/) or [Postman](https://www.postman.com/).
+```shell
+git clone https://github.com/apache/apisix-docker.git
+cd apisix-docker/example
+```
 
-<!--
-#
-#    In addition to the basic Markdown syntax, we use remark-admonitions
-#    alongside MDX to add support for admonitions. Admonitions are wrapped
-#    by a set of 3 colons.
-#    Please refer to https://docusaurus.io/docs/next/markdown-features/admonitions
-#    for more detail.
-#
--->
+Now, you can use `docker-compose` to start APISIX.
+
+<Tabs
+  groupId="cpu-arch"
+  defaultValue="x86"
+  values={[
+    {label: 'x86', value: 'x86'},
+    {label: 'ARM/M1', value: 'arm'},
+  ]}>
+<TabItem value="x86">
+
+```shell
+docker-compose -p docker-apisix up -d
+```
+
+</TabItem>
+
+<TabItem value="arm">
+
+```shell
+docker-compose -p docker-apisix -f docker-compose-arm64.yml up -d
+```
+
+</TabItem>
+</Tabs>
+
+:::note
+
+You can check out [Installing Apache APISIX](./how-to-build.md) for different installation methods.
 
-:::note Note
-If you already have Apache APISIX installed, please skip Step 1, and go to [Step 2](getting-started.md#step-2-create-a-route) directly.
 :::
 
-## Step 1: Install Apache APISIX
+:::info IMPORTANT
 
-You can check out [Building Apache APISIX](./how-to-build.md) for different installation methods.
+Make sure that all the required ports (default: 9080, 9443 and 2379) are available and not used by other system processes.
 
-To get started quickly, we will install Apache APISIX with Docker and enable the [Admin API](./admin-api.md).
+On Unix-based systems, you can run the command below to terminate a process listening on a specific port:
 
 ```bash
-# Download the docker-compose file of Apache APISIX
-git clone https://github.com/apache/apisix-docker.git
-# Switch the current directory to the apisix-docker/example
-cd apisix-docker/example
-# Start Apache APISIX with docker-compose
-docker-compose -p docker-apisix up -d
+sudo fuser -k 9443/tcp
 ```
 
-> Apache APISIX already supports ARM64 architecture. To run Apache APISIX on ARM64, run: `docker-compose -p docker-apisix -f docker-compose-arm64.yml up -d` instead of the last step above.
+If a Docker container is crashing, you can inspect the logs to diagnose the problem:
 
-Please remain patient as it will take some time to download the files and spin up the containers.
+```bash
+docker logs -f --tail container_id
```

Comment:
Should we use something like UPPER_CASE or `${container_id}` here?

```suggestion
docker logs -f --tail container_id
```

Reply:
I have changed it to `<>` could you check?